tutorials/animation/playing_videos.rst

```diff
@@ -0,0 +1,226 @@
+.. _doc_playing_videos:
+
+Playing videos
+==============
+
+Godot supports video playback with the :ref:`class_VideoStreamPlayer` node.
+
+Supported playback formats
+--------------------------
+
+The only supported format in core is **Ogg Theora** (not to be confused with Ogg
+Vorbis audio). It's possible for extensions to bring support for additional
+formats, but no such extensions exist as of July 2022.
+
+H.264 and H.265 cannot be supported in core Godot, as they are both encumbered
+by software patents. AV1 is royalty-free, but it remains slow to decode on the
+CPU and hardware decoding support isn't readily available on all GPUs in use
+yet.
+
+WebM was supported in core in Godot 3.x, but support for it was removed in 4.0
+as it was too buggy and difficult to maintain.
+
+.. note::
+
+    You may find videos with ``.ogg`` or ``.ogx`` extension, which are generic
+    extensions for data within an Ogg container.
+
+    Renaming this file extension to ``.ogv`` *may* allow the video to be
+    imported in Godot. However, not all files with ``.ogg`` or ``.ogx``
+    extensions are videos - some of them may only contain audio.
+
+Setting up VideoStreamPlayer
+----------------------------
+
+1. Create a VideoStreamPlayer node using the Create New Node dialog.
+2. Select the VideoStreamPlayer node in the scene tree dock, go to the inspector
+   and load an ``.ogv`` file in the Stream property.
+
+   - If you don't have your video in Ogg Theora format yet, jump to
+     :ref:`doc_playing_videos_recommended_theora_encoding_settings`.
+
+3. If you wish the video to play as soon as the scene is loaded, check
+   **Autoplay** in the inspector. If not, leave **Autoplay** disabled and call
+   ``play()`` on the VideoStreamPlayer node in a script to start playback when
+   desired.
+
+Handling resizing and different aspect ratios
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default in 4.0, the VideoStreamPlayer will automatically be resized to match
+the video's resolution. You can make it follow usual Control sizing by enabling
+**Expand** on the VideoStreamPlayer node.
+
+To adjust how the VideoStreamPlayer node resizes depending on window size,
+adjust the anchors using the **Layout** menu at the top of the 2D editor
+viewport. However, this setup may not be powerful enough to handle all use
+cases, such as playing fullscreen videos without distorting the video (but with
+empty space on the edges instead). Instead, you can use an
+:ref:`class_AspectRatioContainer` node which is designed to handle this kind of
+use case.
```

Comment:
```suggestion
use case:
```

This change _IMO_ improves flow a little as it makes it more obvious the following explains directly how to do this, but feel free to drop this change if you disagree.